arrocke/gloss-translation
# feat: rich text components

## What has changed

* `RichTextInput` component - uncontrolled rich text input compatible with react-hook-form
* `RichText` component - renders rich text without the ability to edit
* Improvements to use of hebrew and greek fonts
* I added the rich text input to the profile view for testing, so I'll revert that commit once this is approved



## Connected Issues



closes #313 

## QA Steps



Use the rich text input on the profile view to check that each button in the toolbar works.

## Post-Deployment
